src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_ISwitchExpression.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests_Patterns : SemanticModelTestBase
+    {
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Patterns)]
+        [Fact]
+        public void SwitchExpression_Basic()
+        {
+            string source = @"
+using System;
+class X
+{
+    void M(int? x, object y)
+    {
+        y = /*<bind>*/x switch { 1 => 2, 3 => 4, _ => 5 }/*</bind>*/;
+    }
+}
+";
+            string expectedOperationTree = @"
+ISwitchExpressionOperation (3 arms)
```

Comment:
What is the Type of this node, Syntax, etc. Please follow the output pattern for existing nodes.